pandas/core/construction.py

```diff
@@ -412,6 +412,9 @@ def sanitize_array(
 
     # GH#846
     if isinstance(data, np.ndarray):
+        if copy is None:
```

Comment:
So if we ever wanted to reconcile the behavior between ndarray and extension array, we'd change this or L434.

IMO that's not worth the noise.